.github/workflows/check-pg_analytics-schema-upgrade.yml

```diff
@@ -1,155 +1,154 @@
+        run: |
+          sudo apt install clang llvm diffutils
+          cargo install --git https://github.com/zombodb/pg-schema-diff.git
+
+      - name: Extract pgrx Version & Install cargo-pgrx
+        run: |
+          PGRX_VERSION=$(cargo tree --depth 1 -i pgrx -p pg_analytics | head -n 1 | cut -f2 -dv)
+          cargo install -j $(nproc) --locked cargo-pgrx --version ${PGRX_VERSION}
+          cargo pgrx init "--pg${{ env.pg_version }}=/usr/lib/postgresql/${{ env.pg_version }}/bin/pg_config"
+
+          # Save the pgrx version for comparison later
+          echo "FIRST_PGRX_VERSION=${PGRX_VERSION}" >> $GITHUB_ENV
+
+      - name: Generate Schema from this git rev
+        run: cargo pgrx schema -p pg_analytics pg${{ env.pg_version }} > ~/this.sql
+
+      - name: Switch to Base git rev and Generate Schema Again
+        run: |
+          # Switch to the base git rev
+          git checkout .
+          git checkout ${{ github.event.pull_request.base.ref }}
+
+          # See if we need a different cargo-pgrx and install it if so
+          THIS_PGRX_VERSION=$(cargo tree --depth 1 -i pgrx -p pg_analytics | head -n 1 | cut -f2 -dv)
+          if [[ "${THIS_PGRX_VERSION}" != "${FIRST_PGRX_VERSION}" ]]; then
+            # Install cargo-pgrx
+            cargo install -j $(nproc) --locked cargo-pgrx --version ${THIS_PGRX_VERSION} --force
+
+            # Initialize it (again) -- probably unnecessary, but might as well in case ~/.pgrx/config.toml ever changes
+            cargo pgrx init "--pg${{ env.pg_version }}=/usr/lib/postgresql/${{ env.pg_version }}/bin/pg_config"
+          fi
+
+          # Generate schema
+          cargo pgrx schema -p pg_analytics pg${{ env.pg_version }} > ~/old.sql
+
+      - name: Generate Schema Diffs
+        run: |
+          (pg-schema-diff diff ~/old.sql ~/this.sql | grep -v "^$" > ~/diff.sql) || true
+          (diff ~/old.sql ~/this.sql > ~/diff.patch) || true
+
+      - name: Generate Commit Message
+        id: generate_commit_message
+        run: |
+          if test -s ~/diff.sql; then
+            echo "Generating GitHub comment message"
+            {
+              echo 'DIFF<<EOF'
+              echo 'A schema difference was detected.'
+              echo
+              echo 'A suggested "upgrade.sql" script entry might be:'
+              echo
+              echo '```sql'
+              cat ~/diff.sql
+              echo '```'
+
+              #
+              # cargo-pgrx schema doesn't output its generated schema in a stable format
+              # so including the entire diff is garbage.  We still generate it, tho, because
+              # cargo-pgrx will improve someday, and when it does we can uncomment these lines
+              #
+
+              # echo
+              # echo 'The full diff between both schemas is:'
+              # echo
+              # echo '```diff'
+              # cat ~/diff.patch
+              # echo '```'
+              echo EOF
```

Comment:
This we can't uncomment now, since the we upgrade to `0.12.5`

Reply:
I assume you meant we **can** uncomment it so I replaces this with a snippet from the same script in the paradedb repo 👍